python/lsst/dax/data_generator/generator.py

```diff
@@ -55,6 +56,7 @@ def __init__(self, spec):
 
         self.spec = spec
         self.tables = spec.keys()
+        self.timingdict = TimingDict()
 
     @staticmethod
     def _resolve_table_order(spec):
```

Comment:
Just don't see why this internal method has to be a static method, as spec being passed in is self.spec.

Reply:
I don't know, but this is Colin's code and he may have his reasons. I would guess it makes for easier testing, experimenting.